src/modules/game.js

```diff
@@ -0,0 +1,31 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+let gameId;
+
+// this is my Top-Level function
+async function createGame() {
+  const response = await fetch(apiUrl, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ name: 'YOUR_GAME_NAME' }),
+  });
+  const data = await response.json();
+  gameId = data.result;
+}
+
+async function getScores() {
+  const response = await fetch(`${apiUrl}/${gameId}/scores`);
+  const data = await response.json();
+  return data.result;
+}
```

Comment:
- [ ] It will be great to use arrow functions for your functions. This will ensure that you have used the latest javascript syntax in your application as per the requirement. For example: 
```
const createGame = async () => {
}
 ```

Reply:
well resolved! ✔️